app/server/runtime/pom.xml

```diff
@@ -101,6 +101,20 @@
               </excludes>
             </configuration>
           </execution>
+            <execution>
+            <id>default-jar</id>
+            <phase>package</phase>
+            <goals>
+                <goal>jar</goal>
+            </goals>
+            <configuration>
+              <archive>
+                <manifestEntries>
+                  <syndesys-version>${syndesis.version}</syndesys-version>
```

Comment:
typo: syndesys -> syndesis

This value already exists in the `Implementation-Version` manifest header, why do we need additional value?

Reply:
from what I've understood this https://github.com/syndesisio/syndesis/blob/master/app/pom.xml#L36-L37 is the "official global syndesis version" and for whatever the case that might in future differ from the project.version of each component; if that is not the case, fine to read it from `Implementation-Version`.